adb_graphics/default_specs.yml

```diff
@@ -992,6 +1010,7 @@ temp: # Temperature
       pres_sfc:
         hatches: ['', '...']
         levels: [0, 925]
+#    ncl_name: TMP_P0_L105_GLC0 # 2m Temp - wrfnat test
```

Comment:
This one shouldn't be needed since it's included already for both wrfprs and wrfnat under the ua_temp anchor.

Reply:
this was fixed when I added the wrfnat option in create_graphics.